unpackaged/config/config_ldv_org_for_testing/permissionsets/HardDelete.permissionset

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is fine but just so you know that there is also another way:

https://cumulusci.readthedocs.io/en/latest/tasks.html#create-bulk-data-permission-set

Reply:
Didn't know that existed until now.

Reply:
@prescod  I just tried this new task and got the following error:
```
Error: Malformed request https://enterprise-saas-32582-dev-ed.cs68.my.salesforce.com/services/data/v48.0/sobjects/PermissionSet/. Response content: [{'message': "No such column 'PermissionsCreateAuditFields' on sobject of type PermissionSet", 'errorCode': 'INVALID_FIELD'}]
```

Reply:
I didn't read the docs before I linked to them. They say: "NOTE: the org setting to allow Set Audit Fields must be turned on." You can decide for yourself whether that makes it more trouble than its worth in this context.

Reply:
Ah. I'm pretty sure that has to be manually enabled in an org. I'll try using `create_permission_set` and specifying the perm name.

Reply:
That worked perfectly. Definitely better than having to include a perm set in unpackaged.

Reply:
Did I read that the `create_permission_set` command is going to first check for the existence of the perm set before it tries to recreate?